mapsforge-map-android/build.gradle

```diff
@@ -1,16 +1,24 @@
-apply plugin: 'java-library'
+apply plugin: 'com.android.library'
```

Comment:
@Sublimis We do not want such a conversion.

Android module contains only Java code, so it should remain a simple Java module.

Only the tests do not justify such a conversion.
If they are useful, they can exist in their own separate module.

Reply:
No problem to revert. Is there any downside to using com.android.library for an Android library?
Testing for label overlap is very important because it's easy to mess things up (as evidenced by the longevity of the bug).

Reply:
@Sublimis If a library contains only Java code and not Android resources,
there is no real reason to have a more complicated Android aar library.